File: dev-docs/snippets/document-blob/pause-all.ts
import type { JsBaoClient } from "js-bao-wss-client";

export function pauseAll(client: JsBaoClient, documentId: string) {
  // #region example
  const blobs = client.document(documentId).blobs();
  blobs.pauseAll(); // pause every in-progress upload for this document
  // #endregion example
}

File: dev-docs/snippets/document-blob/pause-all.swift
import JsBaoClient

func pauseAll(client: JsBaoClient, documentId: String) {
  // #region example
  let blobs = client.documents.blobs(documentId: documentId)
  blobs.pauseAll()  // pause every in-progress upload for this document
  // #endregion example
}
